### PR TITLE
style: format single-line.zsh to zinit convention

### PR DESCRIPTION
**Pull request by vladdoster**
## Description

Format `share/single-line.zsh` to Zinit's zsh code style convention

## Motivation and Context

Consistency

## Related Issue(s)

#238

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

Signed-off-by: Vladislav Doster <mvdoster@gmail.com>

### CODE REVIEW

**Comment by vladdoster** [comment]
I think this greatly improves grok-ability

File: share/single-line.zsh [hunk 1]
```diff
@@ -1,19 +1,27 @@
 #!/usr/bin/env zsh
+#
+# Copyright (c) 2016-2020 Sebastian Gniazdowski and contributors
+# Copyright (c) 2021-2022 zdharma-continuum and contributors
 
 emulate -R zsh
-setopt extendedglob warncreateglobal typesetsilent rcquotes noshortloops
+setopt extendedglob noshortloops rcquotes typesetsilent warncreateglobal
```
Comment (vladdoster) [inline share/single-line.zsh:7]: sorted alphabetically

File: share/single-line.zsh [hunk 2]
```diff
@@ -1,19 +1,27 @@
...
+
+local IFS= n=$'\n' r=$'\r' zero=$'\0'
```
Comment (vladdoster) [inline share/single-line.zsh:9]: sorted alphabetically

**Review by pschmitt — APPROVED** [review]

**Comment by vladdoster** [comment]
@pschmitt ty 👍🏻

**Pull request merged**